packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -152,92 +260,79 @@ export default function OrgNav({
-                    [activeLink]: activeNav === 'accessManager',
-                  })}
-                >
-                  Access Manager
-                </a>
+                <IconButton ariaLabel="Dropdown">
+                  <Icon glyph={open ? 'CaretUp' : 'CaretDown'} />
+                </IconButton>
               }
+              className={accessManagerMenuContainer}
             >
-              Organization Access Manager
-            </Tooltip>
+              <p className={accessManagerMenuItem}>
+                <strong>Organization Access:</strong> {current.orgName}
+              </p>
+              <p className={accessManagerMenuItem}>
+                <strong>Project Access:</strong> {currentProjectName ?? 'None'}
+              </p>
+            </Menu>
           </li>
           <li role="none" className={supportContainer}>
-            <Tooltip
-              align="bottom"
-              justify="middle"
-              variant="dark"
-              trigger={
-                <a
-                  href={orgNav.support}
-                  className={cx(linkText, {
-                    [activeLink]: activeNav === 'support',
-                  })}
-                >
-                  Support
-                </a>
-              }
+            <LinkElement
+              href={orgNav.support}
+              isActive={activeNav === 'support'}
+              tooltipText="View the Organization Support"
             >
-              Organization Support
-            </Tooltip>
+              Support
+            </LinkElement>
           </li>
           <li role="none">
-            <Tooltip
-              align="bottom"
-              justify="middle"
-              variant="dark"
-              trigger={
-                <a
-                  href={orgNav.billing}
-                  className={cx(linkText, {
-                    [activeLink]: activeNav === 'billing',
-                  })}
-                >
-                  Billing
-                </a>
-              }
+            <LinkElement
+              href={orgNav.billing}
+              isActive={activeNav === 'billing'}
+              tooltipText="View the Organization Billing"
             >
               Billing
-            </Tooltip>
+            </LinkElement>
           </li>
         </ul>
       </div>
       <div>
-        <Tooltip
-          align="bottom"
-          justify="middle"
-          variant="dark"
-          trigger={
-            <a
-              href={orgNav.allClusters}
-              className={cx(rightSideLinkStyle, linkText, {
-                [activeLink]: activeNav === 'allClusters',
-              })}
-            >
-              All Clusters
-            </a>
-          }
+        <LinkElement
+          href={orgNav.allClusters}
+          isActive={activeNav === 'allClusters'}
+          tooltipText="View all clusters across your organizations"
+          className={css`
+            margin-right: 25px;
```

Comment:
Hmm... There's gotta be a better place to put this margin, right?

Reply:
I thought if it was just one rule we were inlining now, but let me know if you want me to move

Reply:
nvm 👍